### PR TITLE
More User Provisioning

### PR DESCRIPTION
**Pull request by turnipdabeets**
Address some feedback after merging and opening up the review to more people. 

To view the rendered version click on `Visit Preview`

### CODE REVIEW

File: docs/workflows/user-provisioning.md [hunk 1]
```diff
@@ -153,3 +153,9 @@ func main() {
 
 </TabItem>
 </Tabs>
+
+### Groups
+
+You can also create groups and add or remove identities from groups in the Beyond Identity API.
+
+For deleting, updating, and managing groups via API, see the [API documentation](https://developer.beyondidentity.com/api/v1#tag/Groups).
```
Comment (BrendanSChang) [inline docs/workflows/user-provisioning.md:161]: Is this duplicated from the section above?

File: docs/workflows/api-token.md [hunk 2]
```diff
@@ -114,7 +114,7 @@ import (
 func main() {
 client := &http.Client{}
 var data = strings.NewReader(`grant_type=client_credentials&scope=SCOPES`)
```
Comment (BrendanSChang) [inline docs/workflows/api-token.md:116]:
```suggestion
data := strings.NewReader(`grant_type=client_credentials&scope=${SCOPES}`)
```

Similar changes for the Golang code snippets elsewhere too.

And should all the variables in all the code snippets be updated to use `${}`? E.g.`${TENANT_ID}`?

**Review by BrendanSChang — CHANGES_REQUESTED** [review]

File: docs/workflows/user-provisioning.md (hunk 1 above)
Comment (turnipdabeets) [reply docs/workflows/user-provisioning.md:161]: There's a section above to the console, but this is new for the API to separate that weird sentence that was joined with creating an identity

File: docs/workflows/api-token.md (hunk 2 above)
Comment (turnipdabeets) [reply docs/workflows/api-token.md:116]: I might make a separate MR for `${}` because there's a lot of other files that we would need to update too.

Comment (BrendanSChang) [reply docs/workflows/api-token.md:116]: Got it, are the Golang snippet changes (to use the shorthand variable declaration `:=`) going to be in this MR?

Comment (turnipdabeets) [reply docs/workflows/api-token.md:116]: Probably change all code snippet stuff in another MR. We're also looking into finding a component that would automatically generate snippets for us from a curl. If you have any ideas, open to hearing them!

Comment (BrendanSChang) [reply docs/workflows/api-token.md:116]: Sounds good! I don't have any prior experience with any tool like that but I'll keep an eye out.

**Review by BrendanSChang — APPROVED** [review]

**Pull request merged**